pkg/cmd/server/api/helpers.go

```diff
@@ -0,0 +1,129 @@
+package api
+
+import (
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+
+	kclient "github.com/GoogleCloudPlatform/kubernetes/pkg/client"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd"
+
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/server/crypto"
+)
+
+func GetKubeClient(kubeConfigFile string) (*kclient.Client, *kclient.Config, error) {
+	parsedKubeConfig, err := clientcmd.LoadFromFile(kubeConfigFile)
```

Comment:
We'll get that working with https://github.com/GoogleCloudPlatform/kubernetes/pull/5541.  For now, the generated .kubeconfig files are self contained.